src/utils/misc.js

```diff
@@ -4,6 +4,10 @@ import createDOMPurify from 'dompurify'
 
 const DOMPurify = createDOMPurify(window)
 
+Handlebars.registerHelper('odd', function(conditional) {
+  return conditional % 2;
+});
```

Comment:
Check if css can be used instead of this. https://tailwindcss.com/docs/hover-focus-and-other-states#odd-child